pkg/minikube/cluster/cluster.go

```diff
@@ -421,6 +422,47 @@ func MountHost(api libmachine.API, path string) error {
 	return nil
 }
 
+// GetVMHostIP gets the ip address to be used for mapping host -> VM and VM -> host
+func GetVMHostIP(host *host.Host) (net.IP, error) {
+	switch host.DriverName {
+	case "kvm":
+		return net.ParseIP("192.168.42.1"), nil
+	case "hyperv":
+		re := regexp.MustCompile("\"VSwitch\": \"(.*?)\",")
+		// TODO(aprindle) Change this to deserialize the driver instead
+		hypervVirtualSwitch := re.FindStringSubmatch(string(host.RawDriver))[1]
+		ip, err := getIPForInterface(fmt.Sprintf("vEthernet (%s)", hypervVirtualSwitch))
+		if err != nil {
+			return []byte{}, errors.Wrap(err, "Error getting VM/Host IP address")
+		}
+		return ip, nil
+	case "virtualbox":
+		ip, err := getIPForInterface("vboxnet0")
```

Comment:
The virtualbox network is not always vboxnet0.  If you had a vboxnet0 before you installed minikube, you'll end up with vboxnet1 (or higher)